### ISSUE TITLE
Why is the Modern Script Editor (react-script-editor sample) still not available as a web part?

### ISSUE THREAD

**Issue by paparush** [pnp/sp-dev-fx-webparts#1744]
## Sample: 
react-script-editor sample

## Authors: 
@mikaelsvenson

## Question:  
Why is the Modern Script Editor web part not appearing in the list of web parts?

## Details:  
I am trying to deploy the web part to a SharePoint 365 site. In the tenant admin console, I have set Allow Custom Scripts to TRUE. That was some 36 hours ago, so I feel I should be beyond the 24-hour window for changing that setting to take effect.

I have edited the ScriptEditorWebPart.manifest.json and set
"requiresCustomScript": false

This is the only line of code in the solution that I have changed. Everything else is straight from this repo that I cloned.

I have successfully added the sppkg to the tenant's App Catalog. Likewise, I successfully add the app to the site.

Unfortunately, the web part is not in the list of available web parts when I edit a modern site page.

Have I missed something? Any assistance in resolving this issue is greatly appreciated.

**Comment by Abderahman88** [comment]
@paparush 

See https://docs.microsoft.com/en-us/answers/questions/176799/unable-to-add-modern-script-editor-inside-our-mode.html

Have you ran these commands in PowerShell ?

**Comment by paparush** [comment]
Thank you for reaching out.
Please clarify, are those two Powershell commands necessary in addition to
going into the SharePoint admin center for my tenant and enabling Custom
Scripting which I did 5 days ago?

[image: image.png]

**Comment by paparush** [comment]
Abderahman,
I followed the step by step instructions in the video linked in the article
you referenced above; following each step exactly as the video instructs,
including the 2 powershell commands.  I'm still unable to see the Modern
Script Editor web part in the list of available web parts when I edit a
modern page in my SharePoint 365 site.

**Comment by Abderahman88** [comment]
I have Modern Script Editor v1.0.0.16 installed in my tenant app catalog and installed the app on a modern teamsite.
It works fine, but I'm not sure if I did something to get it working.

![modernScript](https://user-images.githubusercontent.com/36161889/109882674-8b4f0c80-7c7a-11eb-9375-444c0507254a.PNG)

**Comment by paparush** [comment]
Hello Abderahman,
I wanted to follow up and tell you that I finally got this to work and it
was probably lack of experience on my part.

I had been trying to deploy the Modern Script Editor to the root site of my
sharepoint tenant, such that I was trying to use it from a site page at
this location:
https://tenant-name.sharepoint.com

This never was successful. I could never see the SPFx in the list of
available web parts.

HOWEVER

When I added the app to a site page at this address
https://tenant-name.sharepoint.com/sites/Test_Site
The web part is in the list of available web parts and I am able to add it
to a site page and use it without error.

There must be some issue that I am unaware of when trying to deploy to the
root site.

Thank you again for your help and the great video.

Rush
